tests/ops/functions/test_generator.py

```diff
@@ -176,6 +176,18 @@ class DeprecatedClassOp(CustomOp):
         assert gen.name == "Hermitian"
         assert gen.wires.tolist() == ["a"]
 
+    def test_generator_property_old_default(self):
+        """Test that if the old-style generator property is the old default,

```

Comment:
(Perhaps intentional, if so, ignore this suggestion)
```suggestion
        """Test that if the old-style generator property is the default,
```